src/client/renderingengine.h

```diff
@@ -113,6 +120,14 @@ class RenderingEngine
 		return m_device->run();
 	}
 
+	static ShadowRenderer *get_shadow_renderer()	
```

Comment:
i'm working on removing the global Renderer singleton. Can you make this function non static & call it properly from other objects ?